source/GameAction.cpp

```diff
@@ -214,6 +216,17 @@ void GameAction::LoadSingle(const DataNode &child, const string &missionName)
 			GameData::Missions().Get(toFail);
 		}
 	}
+	else if(key == "flagship add")
+	{
+		if(child.Token(1) == "hardpoints" && child.HasChildren())
+			for(const DataNode &grand : child)
+				hardpoints.emplace_back(grand);
+		else if(child.Token(1) == "attributes" && child.HasChildren())
+		{
+			attributes.emplace_back();
+			attributes.end()->Load(child);
```

Comment:
```suggestion
			attributes.back().Load(child);
```

`end()` is the end iterator and doesn't point to a valid object